### PR TITLE
Adding gherkin scenarios for Banners

### PR DESCRIPTION
**Pull request by dsmewara**
#### Summary of Changes

Created test case scenarios for banners.

### CODE REVIEW

**Comment by puneet0191** [comment]
Thanks, Good Work here :+1:

File: tests/acceptance/banner.feature [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Feature: Banner
+  In order to manage Banner article in the web
+  I need to create modify trash archived Check-In And publish and Unpublish Banner 
+
+  Background:
+    Given Joomla CMS is installed
```
Comment (javigomez) [inline tests/acceptance/banner.feature]: Thanks for the pull @dsmewara ^_^

This given needs to be removed as agreed at https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/54#discussion_r71760814.

File: tests/acceptance/banner.feature [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    Then  I should see the "1 banner successfully trashed." message
+
+   Scenario: Remove trash a Banner
+    Given There is a Banner listing page
+    When I select the Banner with Name "Gsocbanner" which needs to be Remove Trash 
+    And I Remove Trash the Banner
+    Then  I should see the "1 banner successfully deleted." message
+
+  
+
```
Comment (pritalpatel) [inline tests/acceptance/banner.feature:65]: Please remove extra space as there should be only one empty line at end of the file.

File: tests/acceptance.suite.dist.yml [hunk 3]
```diff
@@ -38,4 +38,5 @@ gherkin:
             - Step\Acceptance\Administrator\Login
             - Step\Acceptance\Administrator\User
             - Step\Acceptance\Site\UsersFrontend
-            - Step\Acceptance\Administrator\Content
\ No newline at end of file
+            - Step\Acceptance\Administrator\Content
+            - Step\Acceptance\Administrator\Banner
```
Comment (pritalpatel) [inline tests/acceptance.suite.dist.yml:42]: Please add one empty line at the end of the file.

**Comment by yvesh** [comment]
@dsmewara can you please look at the suggestions from Prital and Puneet? Also your PR has merge conflicts now. Thank you for your work 👍

**Comment by Kubik-Rubik** [comment]
Thank you @dsmewara! We remade your PR at the Automated Testing Code Sprint: https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/94

**Pull request closed without merging**